### PR TITLE
[WIP] Initial version

### PR DESCRIPTION
**Pull request by andygnewman**
To Do;

- [x] abstract out the correct CSS styles
- [x] improve naming conventions

To Do in other repos;

next-article (https://github.com/Financial-Times/next-article/pull/1232)

- [x] update class names based on new naming conventions (in XSLT transforms)
- [x] update transforms tests
- [x] add n-content-body to bower / import CSS
- [x] delete overlapping CSS

next-stream-page (https://github.com/Financial-Times/next-stream-page/pull/894)

- [x] add n-content-body to bower / import CSS
- [ ] delete overlapping CSS

Deployment process

1. Launch n-content-body
2. Launch next-stream-page with n-content-body added, but don't delete the overlapping CSS
3. Launch next-article with all changes
4. Launch next-stream-page with deleted overlapping CSS

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "**/.*",
+    "node_modules",
+    "bower_components"
+  ],
+	"main": [
+		"main.scss"
+	],
+	"license": "MIT",
+	"dependencies": {
+				"next-sass-setup": "^7.0.0"
```
Comment (i-like-robots) [inline bower.json]: Is this needed for anything? Maybe for a demo/test page?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "n-content-body",
+  "version": "0.0.0",
+  "devDependencies": {
```
Comment (i-like-robots) [inline package.json]: Is this needed for anything? Maybe for a demo/test page?

File: scss/_body.scss [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+	&--serif {
+		@include oTypographySerif(m);
+	}
+
+	> p {
+		margin: 0.3em 0 0.8em;
+	}
+
+	> *:first-child {
```
Comment (i-like-robots) [inline scss/_body.scss]: universal selector is implied

File: scss/_body.scss [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+		}
+	}
+
+	strong {
+		font-weight: 600;
+	}
+
+	// COPYRIGHT
+
+	.copyright-notice {
```
Comment (i-like-robots) [inline scss/_body.scss]: Maybe `.n-content-copyright` at the top-level so it's not scoped to be used outside main body

File: scss/_body.scss [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+	blockquote {
+		color: getColor('cold-1');
+		position: relative;
+
+		p {
+			@include oTypographySerifDisplay(m);
+		}
+
+		.quote--full {
+			padding-top: 32px;
```
Comment (i-like-robots) [inline scss/_body.scss]: `&.quote--full`? Good use case for a demo page

File: scss/_body.scss [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+					left: 16px;
+					content: "";
+				}
+			}
+
+		}
+	}
+
+	// DATA TABLES
+	.data-table {
```
Comment (i-like-robots) [inline scss/_body.scss]: I think we can get away with simply `table`

File: scss/_body.scss [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+	.subhead--crosshead {
+		border-bottom: 1px solid black;
+	}
+	.subhead--standard {
+		@include oTypographySansBold(m);
+	}
+
+	// TAGS
+
+	.tags__list--inline {
```
Comment (i-like-robots) [inline scss/_body.scss]: Do you know where the `.tags` and `.tags__list` classes are defined?

File: scss/_body.scss [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+		list-style-type: none;
+		padding-left: 0;
+	}
+
+	.tag--inline {
```
Comment (i-like-robots) [inline scss/_body.scss]: Same as above, would be good to clean up orphaned bits

File: scss/_tags.scss [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+.n-content-tag__link {
+	border-bottom: 1px dotted transparent;
+	transition-property: border-bottom-color, color;
+	transition-duration: 0.3s;
+	&:hover {
+		border-bottom-color: darken(getColorFor(topic, text), 20%);
+	}
+}
+
+.n-content-tag__with-follow {
```
Comment (i-like-robots) [inline scss/_tags.scss]: `--with-follow` (variation rather than element)?

File: scss/_tags.scss [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+	@include nLinksTopic;
+	font-style: normal;
+}
+
+.n-content-tag--medium {
+	@include oTypographySansBoldSize(m);
+}
+
+.n-content-tag__link {
+	border-bottom: 1px dotted transparent;
```
Comment (i-like-robots) [inline scss/_tags.scss]: Can we use `nLinksTopic` mixin?

File: scss/_utilities.scss [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+.n-content-u-border--left {
+	border-left: 1px solid getColor('warm-3');
+}
+
+.n-content-u-align-center {
+	text-align: center;
+}
+
+//Want this to be capital L for consistency with o-grid definition
+.n-content-u-hide--L { 	// scss-lint:disable SelectorFormat
```
Comment (i-like-robots) [inline scss/_utilities.scss]: Can't find any instances of this being used

Comment (i-like-robots) [inline scss/_utilities.scss]: We should provide hide/show utils for all breakpoints or none at all. I would probably go for the latter so it's entirely up to the implementation.

File: scss/_body.scss (hunk 6 above)
Comment (andygnewman) [inline scss/_body.scss]: data-table is a specific methode / capi thing that is different from a standard table, so would suggest sticking with data-table.

File: package.json (hunk 2 above)
Comment (andygnewman) [inline package.json]: next-build-tools is required for bottling the release so included to kick that process off
plus have put a verify in using obt.

File: scss/_body.scss [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+		p {
+			@include oTypographySerifDisplay(m);
+		}
+
+		.quote--full {
+			padding-top: 32px;
+			padding-bottom: 0;
+
+			p:first-child {
+				&:before {
```
Comment (i-like-robots) [inline scss/_body.scss]: Doesn't need nesting in `p:first-child`

File: scss/_body.scss [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+		}
+		thead,
+		tbody {
+			&:before {
+				content: "";
+				display: block;
+				height: 10px;
+			}
+		}
+		tbody tr:nth-child(2n+1) {
```
Comment (i-like-robots) [inline scss/_body.scss]: `nth-child(odd)`

**Pull request merged**